www/src/components/mdx/code-block/styles.js

```diff
@@ -8,7 +8,7 @@ const previewThemes = {
 
 const classes = {
     // Border color is defined by the theme above.
-    preview: `pa4 mb1 ba bw-2 br2 overflow-x-auto ${styles.preview}`,
```

Comment:
This change removes the `x` overflow from the React live examples. It's needed because `react-waypoint`, the tool used to determine if an element is visible, used the element's first scrollable parent as the reference. This means that if the element is visible within the scrollable, then `react-waypoint` runs the `onEnter` function.

This default behavior is desired in most cases, but caused problems in our case because the elements were _always_ visible within the live previews.

We could provide a prop to control [React Waypoint's `scrollableAncestor` prop](https://github.com/brigade/react-waypoint) and use it here, but we're probably better off just sticking to the defaults until someone requests it.